srcpkgs/booster/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'booster'
+pkgname=booster
+version=0.3
+revision=1
+build_style=go
+go_import_path=github.com/anatol/booster
+go_package="${go_import_path}/init ${go_import_path}/generator"
+go_mod_mode=readonly
+hostmakedepends="git"
+depends="busybox-static"
```

Comment:
From the arch package, it would seem busybox is a dependency only for the rescue shell, and bash is a strict dependency...

Why did you go with -static, too?

Reply:
I'm not yet sure how bash is a dependency (Arch uses `#!/bin/bash` instead of `#!/bin/sh` for the kernel hooks but that's all I can see so far).

I would say rescue shell is core functionality, and Arch's busybox package is -static.

Reply:
Fair enough. It would be nice to have a busybox virtual, if this can work with non static one.